CONTRIBUTING.md

```diff
@@ -72,25 +72,18 @@ will be easily spotted in pull request. It will also ensure that OpenTelemetry
 doesn't expose APIs outside of the library primary concerns like a generic
 helper methods.
 
-#### How to enable and configure
+#### How to configure
 
 * Create a folder in your project called `.publicApi` with the frameworks that
   as folders you target.
 * Create two files called `PublicAPI.Shipped.txt` and `PublicAPI.Unshipped.txt`
    in each framework that you target.
-* Add the following lines to your csproj:
-
-```xml
-<ItemGroup>
-  <AdditionalFiles Include=".publicApi\$(TargetFramework)\PublicAPI.Shipped.txt" />
-  <AdditionalFiles Include=".publicApi\$(TargetFramework)\PublicAPI.Unshipped.txt" />
-</ItemGroup>
-```
-
 * Use
    [IntelliSense](https://docs.microsoft.com/visualstudio/ide/using-intellisense)
    to update the publicApi files.
 
+Enabled by default, add `<RunApiCompat>false</RunApiCompat>` to your csproj to disable.
```

Comment:
any reason why we want to allow this be disabled? Its probably a good thing in contrib repo, but this repo should not need such bypass options. 
If this is for temporarily bypassing api check while developing locally, be explicit about it.

Reply:
Just calling it out `how` to, I can add a more explicit comment about `why` and `when`